examples/cloudaccounts/gcp/gcpcloudaccount.md

```diff
@@ -100,3 +109,5 @@ func main() {
 	fmt.Printf("response type: %T\n Content: %+v\n", OrganizationalUnitIDResponse, OrganizationalUnitIDResponse)
 
 }
+
+```
```

Comment:
in the get specific cloud account you write getAll, please change it to get function that gets as input query parameter

Reply:
Please add delete function